pkg/nvml/api.go

```diff
@@ -576,6 +576,12 @@ type EventSet interface {
 	Free() Return
 }
 
+type GpmSample interface {
+	Free() Return
+	Get(Device Device) Return
+	MigGet(Device Device, GpuInstanceId int) Return
```

Comment:
Nit:
* `Device Device` -> `Device` or `device Device`
* `GpuInstanceId int` -> `int` or `gpuInstanceID int`.

Reply:
Yes, I am doing a pass now to lowercase all variable names.

Reply:
We can do it as a follow-up if that gets too much.

Reply:
I have made this change, updated, and moved the PR out of draft.